editor/src/uuiui/styles/css-vars.tsx

```diff
@@ -0,0 +1,30 @@
+import { UtopiColor } from './utopi-color-helpers'
+// TODO: pass in colorTheme to updateCssVars from a hook and remove the dependency here:
+import {
+  colorThemeCssVariables,
+  useColorTheme,
+  UtopiaTheme,
+  // utopiaThemeCssValues
+} from './theme'
+import React from 'react'
+
+export const ColorThemeStyleComponent = React.memo(() => {
+  const colorTheme = useColorTheme(),
+    { inverted, ...currentTheme } = colorTheme
+
+  return (
+    <style>
+      {':root {'}
+      {Object.entries(colorThemeCssVariables).map((entry) => {
+        const [key, variable] = entry
+        const cssValue = currentTheme[key as keyof typeof currentTheme].cssValue
+        return `${variable}:${cssValue};`
+      })}
+      {/* {Object.entries(utopiaThemeCssValues).map((entry) => {
+        const [variable, value] = entry
+        return `${variable}:${value};`
+      })} */}
```

Comment:
Please can you remove the commented code from this PR completely. We don't really gain anything from having the code committed and commented out, especially if it's going to be uncommented (and quite possibly changed / moved) in another PR following shortly after this.